tests/settings.py

```diff
@@ -56,6 +56,18 @@
             }
         }
     }
+elif os.environ.get('IMPORT_EXPORT_TEST_TYPE') == 'postgres':
+    IMPORT_EXPORT_USE_TRANSACTIONS = True
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql',
+            'NAME': 'import_export',
+            'USER': 'postgres',
+            'PASSWORD': 'postgres',
+            'HOST': 'localhost',
```

Comment:
Hmm, when I try that I get 'django.db.utils.OperationalError: fe_sendauth: no password supplied'

I guess you could use environment variables to pass in user / pw like you do for the mysql case.